src/index.js

```diff
@@ -304,12 +304,6 @@ export function configure(env) {
 	};
 
 	if (watch) {
-		webpackConfig.entry = [
```

Comment:
just curiosity, why we did we remove this?

Reply:
With latest version of webpack, we were getting error that webpack/hot/dev-server was not found, and I saw there was no impact after removing it

Reply:
aah okay!